src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private final Map<String, Integer> fruitsAmount;
+
+    Storage() {
+        fruitsAmount = new HashMap<>();
+    }
+
+    Map<String, Integer> getFruitsAmount() {
```

Comment:
I agree that you can address static public field directly and you don't need this method